prac_05/word_occurrences.py

```diff
@@ -0,0 +1,19 @@
+import operator
+
+text = input("Enter a string: ")
+# Split the string into words
+words = text.split()
+counts_words = {}
+for word in words:
+    if word in counts_words:
+        counts_words[word] += 1
+    else:
+        counts_words[word] = 1
+
+sorted_counts_words = sorted(counts_words.items(), key=operator.itemgetter(0))
```

Comment:
Why do you need operator? You could just use sorted(counts_words) and that's enough